x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_list.tsx

```diff
@@ -98,465 +98,482 @@ const StyledEuiCodeBlock = euiStyled(EuiCodeBlock).attrs({
+          truncateText: true,
+          render: (startedAt: ActionDetails['startedAt']) => {
+            return (
+              <FormattedDate
+                fieldName={TABLE_COLUMN_NAMES.time}
+                value={startedAt}
+                className="eui-textTruncate"
               />
-            </EuiToolTip>
-          );
+            );
+          },
         },
-      },
-      {
-        field: 'createdBy',
-        name: TABLE_COLUMN_NAMES.user,
-        width: hideHostColumn ? '21%' : '14%',
-        truncateText: true,
-        render: (userId: ActionDetails['createdBy']) => {
-          return (
-            <StyledFacetButton
-              icon={
-                <EuiAvatar
-                  name={userId}
-                  data-test-subj={getTestId('column-user-avatar')}
-                  size="s"
+        {
+          field: 'command',
+          name: TABLE_COLUMN_NAMES.command,
+          width: hideHostColumn ? '21%' : '10%',
+          truncateText: true,
+          render: (_command: ActionDetails['command']) => {
+            const command = getCommand(_command);
+            return (
+              <EuiToolTip content={command} anchorClassName="eui-textTruncate">
+                <FormattedMessage
+                  id="xpack.securitySolution.responseActionsList.list.item.command"
+                  defaultMessage="{command}"
+                  values={{ command }}
                 />
-              }
-            >
-              <EuiToolTip content={userId} anchorClassName="eui-textTruncate">
+              </EuiToolTip>
+            );
+          },
+        },
+        {
+          field: 'createdBy',
+          name: TABLE_COLUMN_NAMES.user,
+          width: hideHostColumn ? '21%' : '14%',
+          truncateText: true,
+          render: (userId: ActionDetails['createdBy']) => {
+            return (
+              <StyledFacetButton
+                icon={
+                  <EuiAvatar
+                    name={userId}
+                    data-test-subj={getTestId('column-user-avatar')}
+                    size="s"
+                  />
+                }
+              >
+                <EuiToolTip content={userId} anchorClassName="eui-textTruncate">
+                  <EuiText
+                    size="s"
+                    className="eui-textTruncate eui-fullWidth"
+                    data-test-subj={getTestId('column-user-name')}
+                  >
+                    {userId}
+                  </EuiText>
+                </EuiToolTip>
+              </StyledFacetButton>
+            );
+          },
+        },
+        // conditional hostname column
+        {
+          field: 'agents',
+          name: TABLE_COLUMN_NAMES.host,
+          width: '20%',
+          truncateText: true,
+          render: (agents: ActionDetails['agents']) => {
+            // TODO: compute host names later with hostMetadata? (using agent Ids for now)
+            const hostname = agents?.[0] ?? '';
```

Comment:
just so that we account for multiples - can you instead do a `agents?.join(', ') ?? ''` here ?